scripts/publish.ts

```diff
@@ -0,0 +1,55 @@
+import npmPublish from '@jsdevtools/npm-publish';
+import { Results } from '@jsdevtools/npm-publish/lib/results';
+import fs from 'fs';
+import path from 'path';
+
+const tag = 'dev';
+const dryRun = false;
+const token = process.env.NPM_TOKEN;
+
+/**
+ * Provide a feedback message after a package is published to NPM
+ * @param data
+ */
+const afterPublish = (data: Results) => {
+  if (data.oldVersion !== data.version) {
+    console.log(` ✓ Package "${data.package}" published: ${data.oldVersion} → ${data.version} (${data.tag})\n`);
+  } else if (data.oldVersion === data.version) {
+    console.log(` ✓ Package "${data.package}" is already published: ${data.version} (${data.tag})\n`);
+  } else {
+    console.log(data, '\n');
+  }
+};
+
+(async () => {
+  // Start by publishing the main romcal library
+  console.log(` - Publishing romcal`);
+  const data = await npmPublish({
+    package: path.join(__dirname, '..', 'package.json'),
```

Comment:
```suggestion
    package: path.join(__dirname, '../package.json'),
```

Reply:
It's more performant to keep them separated, but ok for the readability.

Reply:
I didn’t know it is more performant when we merge three (more) strings instead of two (less) strings. Why is it so?

If it is more performant, I think we should them separated.